spec/controllers/users_controller_spec.rb

```diff
@@ -30,28 +30,26 @@
     include ERB::Util
     RSpec.shared_examples 'changing particular mailer settings' do
       it 'can be enabled in settings' do
-        student.update!(setting => false)
+        student.update!(receives_invite_emails: false)
```

Comment:
I'm not sure why you made these changes, but the point of this code is to allow us to use the same examples with multiple different settings. You should revert these changes (and the same for `settings` down below).